app/controllers/user.js

```diff
@@ -0,0 +1,20 @@
+const { user } = require('../models'),
+  logger = require('../logger');
+
+const save = (request, response) => {
+  user
```

Comment:
add return or delete brackets

Reply:
In this case I want to return `undefined`, I mean, there's no need to return anything in this case. Should I always return something in the functions?

Reply:
I want you to get the habit of returning promises. In this case, nothing will happen, because you have only one block of code which is the promise itself, and it will end the request with the response. But if you have another block of code and you call the promise without returning it, you will not know when it finally resolves to call the response. The cost of returning every promise is 0 (literally just deleting the brackets) and in my experience is the best practice to avoid race conditions